uefi-test-runner/src/proto/network/snp.rs

```diff
@@ -34,7 +34,8 @@ pub fn test() {
             .initialize(0, 0)
             .expect("Failed to initialize Simple Network");
 
-        simple_network.reset_statistics().unwrap();
+        let res = simple_network.reset_statistics();
```

Comment:
We should drop a comment here. Something like

`// virtio-net driver in OVMF is expected to not support this`